src/components/ErrorContent/ErrorContent.module.scss

```diff
@@ -1,33 +1,54 @@
-@import '../../../node_modules/@santiment-network/ui/mixins';
+@import "../../../node_modules/@santiment-network/ui/mixins";
 
 .container {
   display: flex;
-  max-width: 1152px;
-  margin: auto;
   align-items: center;
+  justify-content: center;
+  flex: 1;
+  gap: 40px;
+}
 
-  @include responsive('phone', 'phone-xs', 'tablet', 'laptop') {
-    flex-direction: column-reverse;
-    padding: 16px;
-  }
+:global(.laptop) .container,
+:global(.phone) .container,
+:global(.tablet) .container,
+:global(.phone-xs) .container {
```

Comment:
- `[D-0-0]` To improve readability we can sort the devices in ascending size order, e.g: `phone-xs, phone, tablet, ...`
- `[D-0-1]` To decrease code repetition and make future refactoring easier we can use the scss's parent selector `&` like this:
```scss
.container {
  :global(.phone-xs) &, :global(.phone) { ... }
}
```